tests/test_timeslices.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+
+
+def test_timeslices(test_mp):
+    timeslices = test_mp.timeslices()
+    obs = timeslices[timeslices.category == 'COMMON']
+    assert all(obs.columns == ['name', 'category', 'duration'])
```

Comment:
Please add a comment with a plain language / human-readable readable description of what behaviour is being tested in each assertion.

Reply:
Could you please check if added comments correspond to your expectations?